pkg/oss/utils.go

```diff
@@ -99,6 +99,16 @@ func parseOtherOpts(otherOpts string) (mountOptions []string, err error) {
 	return mountOptions, nil
 }
 
+func getPodUid(targetPath, volumeID string) (podUid string) {
```

Comment:
See https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#csidriverspec-v1-storage-k8s-io (podInfoOnMount) for how to get pod info. Maybe we should not rely on the format of targetPath.

Reply:
it seems that only NodePublishVolumeRequest  has related volume_context of podInfoOnMount. But we need it on unpublish stage.
BTW, `csi.alibabacloud.com/used-by-pod/namespace/pod-name` is invalid. annotation key could only has ONE "/".

Reply:
OK, then the part before "/" should be a domain name.